engagement_db_to_analysis.py

```diff
@@ -12,13 +12,16 @@
 
     parser.add_argument("--incremental-cache-path",
                         help="Path to a directory to use to cache results needed for incremental operation.")
+
     parser.add_argument("user", help="Identifier of the user launching this program")
     parser.add_argument("google_cloud_credentials_file_path", metavar="google-cloud-credentials-file-path",
                         help="Path to a Google Cloud service account credentials file to use to access the "
                              "credentials bucket"),
     parser.add_argument("configuration_module",
                         help="Configuration module to import e.g. 'configurations.test_config'. "
                              "This module must contain a PIPELINE_CONFIGURATION property")
+    parser.add_argument("membership_group_dir_path",
+                        help="Path to a directory to use to read & write membership group csvs to.")
```

Comment:
I suggest this should be optional and then add a condition after importing the pipeline config module to check if the   membership_group_configuration is not None, if so raise error indicating membership_group_dir_path is required.

Reply:
This introduces a point of failure when we forget to pass membership_group_dir_path which is what we are trying to solve with the new pipelines. Checking for if the config exists before tagging should suffice.